microprofile/jwt-auth/src/main/java/io/helidon/microprofile/jwt/auth/JwtAuthProvider.java

```diff
@@ -1205,6 +1245,17 @@ public Builder decryptKeyLocation(String decryptKeyLocation) {
             return this;
         }
 
+        /**
+         * Required decryption key algorithm.
+         *
+         * @param decryptionKeyAlgorithm required decryption key algorithm
+         * @return updated builder instance
+         */
+        public Builder decryptKeyAlgorithm(String decryptionKeyAlgorithm) {
+            this.decryptionKeyAlgorithm = decryptionKeyAlgorithm;
```

Comment:
Required param non-null?

Reply:
This can be null. I think better would be probably to change the wording to not say required, but expected :-) This required could be a bit misleading.